docs/pages/kubernetes-access/guides/discovery/aws.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: Teleport EKS auto-discovery (Preview)
+description: Auto-discovery of EKS clusters in AWS cloud.
+---
+
+Similarly to [AKS discovery](./azure.mdx), the EKS Discovery can automatically 
+discover any EKS cluster running and enroll it if its tags match the configured labels. 
+
+As mentioned in [Kubernetes Discovery](../discovery.mdx), the discovery process 
+requires two separate Teleport components. The first, `discovery_service`, is 
+responsible for watching the cloud provider and checking if there are any 
+new clusters or if there have been any modifications to previously discovered clusters. 
+The `kubernetes_service` monitors the clusters created by `discovery_service` and 
+is responsible for authenticating into them and forwarding the requests.
+
+<Notice type="tip">
+
+This guide presents the `discovery_service` and `kubernetes_service` running in 
+the same process, however both can run independently and on different machines. To do 
+that, you just need to create a valid token for each of the roles and separate their 
+configurations. Running the processes independently is beneficial if access 
+to different clusters requires `kubernetes_service` to be on the same network as 
+the cluster.
+
+</Notice>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- AWS IAM permissions to list and describe EKS clusters and IAM access to the cluster.
+- EKS clusters running. 
+
+## Known Limitations
+
+Due to the authentication mode that AWS has chosen for EKS clusters, it is not possible 
+for the Teleport process to set up access for itself without previously having access to the cluster.
+This is because the authentication method requires the Teleport IAM Role to be configured 
+in a `configmap/aws-auth` that exists in the `kube-system` namespace.
+
+Please ensure that the permissions are set correctly if the Teleport process does not use 
```

Comment:
Are there some handy commands that an operator can copy/paste to determine this?

Reply:
Yes but nothing simple.

To test that they can:
- Inspect the `aws-auth` config and validate if the Teleport IAM role is there.
- Check if ClusterRole and ClusterRoleBinding exist.

Or:
- Manually create a Token for the IAM role they want to use in Teleport
- Setup a kubeconfig for the cluster
- use `kubectl can-i ...` to check if everything works